test/test_mps.py

```diff
@@ -7829,7 +7829,7 @@ def test_device_synchronize(self):
         x.backward(torch.randn_like(x))
         torch.mps.synchronize()
 
-    @unittest.expectedFailure
+    @serialTest
```

Comment:
```suggestion
    @serialTest()
```

Reply:
Ouch, this makes me think I need to to check if the test is really running